docs/CODING.md

```diff
@@ -73,3 +73,23 @@ and reviewers. This document lists such coding conventions.
   few reasons to use asserts).
 * Any conversion between strings and bytes happens with `os.fsencode/fsdecode`.
 * Messages are to be expressed in strings (log, errors, etc).
+
+## DOCSTRINGS
```

Comment:
Is it possible to enforce this rules using flake8 ? That would greatly simplify the review process as I'm not the best to follow rules :D

Reply:
Not a small task, I have no clue if flake8 can do it, and how to configure it. But interesting idea, I've parked it in an enhancement request #530 (perhaps someone else could take it).

Reply:
+1 I saw the task